From weiyi.huang@gmail.com  Wed Jan 21 09:35:26 2009
From: Huang Weiyi <weiyi.huang@gmail.com>
Date: Sun, 11 Jan 2009 17:22:36 +0800
Subject: Staging: remove duplicated #include's
To: gregkh@suse.de
Cc: linux-kernel@vger.kernel.org
Message-ID: <20090111171449.0AB4.WEIYI.HUANG@gmail.com>


Removed duplicated #include's in
  drivers/staging/altpciechdma/altpciechdma.c
  drivers/staging/comedi/comedidev.h
  drivers/staging/rt2860/rt_linux.h
  drivers/staging/rt2870/rt_linux.h

Signed-off-by: Huang Weiyi <weiyi.huang@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/altpciechdma/altpciechdma.c |    1 -
 drivers/staging/comedi/comedidev.h          |    1 -
 drivers/staging/rt2860/rt_linux.h           |    1 -
 drivers/staging/rt2870/rt_linux.h           |    1 -
 4 files changed, 4 deletions(-)

--- a/drivers/staging/altpciechdma/altpciechdma.c
+++ b/drivers/staging/altpciechdma/altpciechdma.c
@@ -46,7 +46,6 @@
 #include <linux/cdev.h>
 #include <linux/delay.h>
 #include <linux/dma-mapping.h>
-#include <linux/delay.h>
 #include <linux/init.h>
 #include <linux/interrupt.h>
 #include <linux/io.h>
--- a/drivers/staging/comedi/comedidev.h
+++ b/drivers/staging/comedi/comedidev.h
@@ -36,7 +36,6 @@
 #include <linux/mm.h>
 #include <linux/init.h>
 #include <linux/vmalloc.h>
-#include <linux/mm.h>
 #include "interrupt.h"
 #include <linux/dma-mapping.h>
 #include <linux/uaccess.h>
--- a/drivers/staging/rt2860/rt_linux.h
+++ b/drivers/staging/rt2860/rt_linux.h
@@ -65,7 +65,6 @@
 #include <linux/vmalloc.h>
 
 
-#include <linux/wireless.h>
 #include <net/iw_handler.h>
 
 // load firmware
--- a/drivers/staging/rt2870/rt_linux.h
+++ b/drivers/staging/rt2870/rt_linux.h
@@ -65,7 +65,6 @@
 #include <linux/vmalloc.h>
 
 
-#include <linux/wireless.h>
 #include <net/iw_handler.h>
 
 // load firmware
